point-of-sale/src/client/utils/arraysEqual.ts

```diff
@@ -1,5 +1,4 @@
 export function arraysEqual(a: any[], b: any[]) {
-    if (a === b) return true;
```

Comment:
This probably shouldn't be removed, it's a short circuit if the same instance is passed in.